src/mme-app/handlers/service_req.c

```diff
@@ -206,15 +207,23 @@ post_to_next()
 static int
 post_service_reject()
 {
-    struct commonRej_info s1ap_rej;
-	log_msg(LOG_INFO, "Sending Service Rej \n");
-	struct service_req_Q_msg *service_req =
-				(struct service_req_Q_msg *) buf;
-    s1ap_rej.IE_type = S1AP_SERVICE_REJECT;
+    log_msg(LOG_INFO, "Sending Service Rej \n");
```

Comment:
Lets add available User identifier in the logs.

Reply:
done. added mTMSI